pod/perlhack.pod

```diff
@@ -175,7 +175,7 @@ L<perlgit>.
 You will need a copy of Git for your computer.  You can fetch a copy of
 the repository using the git protocol:
 
-  % git clone git://github.com/Perl/perl5.git perl
+  % git clone git@github.com:Perl/perl5.git perl
```

Comment:
Since this is for read access, I believe it should also present the HTTPS URL for ease of access.

Reply:
My mistake, the HTTPS URL is presented below. Perhaps we should just remove the "git protocol" section here and skip to the HTTPS clone option.